tests/kernels/test_cutlass.py

```diff
@@ -190,3 +190,44 @@ def test_cutlass_subset():
                         b.to(dtype=torch.float32)).to(dtype=torch.bfloat16)
 
     assert torch.allclose(out, baseline, rtol=1e-1, atol=1e0)
+
+
+# Test to make sure eager mode works
```

Comment:
This should be `Test to make sure CUDA graph works` (eager mode is the opposite, i.e. without CUDA graphs)

Reply:
🤦